From ac639ece74e4db8cfcd660d790ccd719b13991c9 Mon Sep 17 00:00:00 2001
From: Felix Hammarstrand <felix.hammarstrand@flir.se>
Date: Wed, 30 Oct 2019 16:01:57 +0100
Subject: [PATCH] mxc mipi dsi: move supend order to fix display not entering
 standby

move mipi_dsi  driver suspend after lcdif driver suspend
because mipi_dsi suspend disables mipi phy clock
which lcdif suspend depends on
---
 drivers/video/fbdev/mxc/mipi_dsi_northwest.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/video/fbdev/mxc/mipi_dsi_northwest.c b/drivers/video/fbdev/mxc/mipi_dsi_northwest.c
index 468a025..97b6450 100644
--- a/drivers/video/fbdev/mxc/mipi_dsi_northwest.c
+++ b/drivers/video/fbdev/mxc/mipi_dsi_northwest.c
@@ -1511,8 +1511,8 @@ static int mipi_dsi_resume(struct device *dev)
 }
 
 static const struct dev_pm_ops mipi_dsi_pm_ops = {
-	.suspend = mipi_dsi_suspend,
-	.resume  = mipi_dsi_resume,
+	.suspend_late = mipi_dsi_suspend,
+	.resume_early  = mipi_dsi_resume,
 	.runtime_suspend = mipi_dsi_runtime_suspend,
 	.runtime_resume  = mipi_dsi_runtime_resume,
 	.runtime_idle	 = NULL,
